tests/pytest_tests/system_tests/test_launch/test_job.py

```diff
@@ -33,3 +41,165 @@ def test_job_call(relay_server, user, wandb_init, test_settings):
         assert rqi["runSpec"]["job"].split("/")[-1] == f"job-{docker_image}:v0"
         assert rqi["runSpec"]["project"] == proj
         run.finish()
+
+
+def test_create_job_artifact(runner, user, wandb_init, test_settings):
+    proj = "test-p"
```

Comment:
@KyleGoyette  this is the almost end to end test, we poke the shape of the upgraded job, i guess not the run itself. How can I best do that?

Reply:
Hmm, you could use the relay server to check stuff. But this is solid as is. Sorry I missed that